src/lamplib/src/genny/toolchain.py

```diff
@@ -195,7 +195,7 @@ def _can_ignore(self):
         )
 
     def _check_toolchain_githash(self):
-        res = "".join(
-            run_command(cmd=["git", "rev-parse", "HEAD"], cwd=self.result_dir, check=True).stdout
-        )
-        return res.strip() == ToolchainDownloader.TOOLCHAIN_GIT_HASH
+        if os.path.exists(self.toolchain_git_hash_file):
```

Comment:
Won't this only check whether there is a git hash file in the first place, without checking that it's the right hash?